src/view.c

```diff
@@ -857,51 +906,250 @@ view_on_output_destroy(struct view *view)
 	view->output = NULL;
 }
 
+#define VIEW_GET_NEXT_EDGES(_view_near_def, _view_far_def, _backward)	\
+	struct output *output = view->output;				\
+	struct server *server = output->server;				\
+	struct view *v;							\
+	wl_list_for_each(v, &server->views, link) {			\
+		if (v == view || v->output != output)			\
+			continue;					\
```

Comment:
There might be some more views to exclude here?

Reply:
Yes, at least
- `view->minimized`
- `view->scene_tree->node.parent != view->server->workspace_current->tree && !view_is_always_on_top(view)`

Reply:
I also exclude `view->minimized || view->maximized` in 0e27cb510f100fd0943ba9a40f8ae0a64c699d2a.

Why would we want to ignore the always-on-top windows? I would rather exclude always-on-bottom, no?

```
view->scene_tree->node.parent != view->server->workspace_current->tree && !view_is_always_on_top(view)
```
^^^ This statement does not work (and I don't understand what  it is doing, leaving it out for now :-/

Reply:
> Why would we want to ignore the always-on-top windows? I would rather exclude always-on-bottom, no?

Windows should be ignored that are *neither*
- part of the current workspace
- part of the always-on-top tree

If one of these conditions is true (and the other criteria matches, e.g. not minimized, ..) the windows should be used.
Basically this is for ignoring windows that are on another workspace.

Reply:
Fixed, see: https://github.com/labwc/labwc/pull/1005/files#r1285129442